src/components/patch-assignment-form/components/rows.tsx

```diff
@@ -52,11 +52,7 @@ export const TableRow = ({
   const [validationError, setValidationError] = useState<string | null>(null);
 
   function onConfirmButtonClick() {
-    if (!newOfficerName || !newOfficerEmail) {
-      setValidationError("Please enter a name and email address");
-      return;
-    }
-    if (!newOfficerEmail.includes("@hackney.gov.uk")) {
+    if (newOfficerEmail && !newOfficerEmail.includes("@hackney.gov.uk")) {
       setValidationError("Email address must include @hackney.gov.uk");
```

Comment:
A future enhancement can be that we add this error message to reference data ES so that it can be reusable. :)

Reply:
Sounds fair! :+1: